README-PTBR.md

```diff
@@ -0,0 +1,101 @@
+# Implementação RSA <a align="right"><img align="right" src="https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54" alt="python"></a>
+
+
+> Implementação simples da lógica  RSA usando puro Python 🔑
+
+<p align="center">
+  <a href="#conceito">Conceito</a> • 
+  <a href="#instalação-e-requisitos">Instalação e Requisitos</a> • 
+  <a href="#como-executar">Como executar</a> •  
+  <a href="#testes">Testes</a> • 
+  <a href="#licença">Licença</a> •
+  <a href="README.md">EN</a>
+</p>
+
+## Conceito
+
+O algoritmo RSA se trata de uma criptografia assimétrica; isso significa que ele usa uma chave pública e uma chave privada (ou seja, duas chaves diferentes, matematicamente ligadas). Como seus nomes sugerem, a chave pública é pubicamente compartilhada, enquanto a chave privada é secreta e não deve ser compartilhada com ninguém.
+
+O algoritmo RSA tem o nome daqueles que o inventaram em 1978: ROn Rivest, Adi Shamir e Leonard Adleman.
+
+
+### Gerando as chaves
+
+1. Selecione dois numeros primos grandes, x e y. Os números primos precisam ser grandes para que seja difícil para alguém decifrá-los.
+2. Calcule n = x * y
+3. Calcule a função totiet: ϕ(n) = (x−1)*(y−1)
+4. Selecione um inteiro *e*, de tal forma que *e* seja co-primo de ϕ(n) e 1 < *e* < ϕ(n). O par de números (n,*e*) constitui a chave pública..
+> Dois inteiros são primos se o único inteiro positivo que os divide for 1.
+
+5. Calcule `d` de modo que `e.d = 1 mod ϕ(n)`
+
+**TRANSLATE HERE: Add can be found using the extended euclidean algorithm. The pair (n,d) makes up the private key.**
+
```

Comment:
Sorry @thalees , what did you mean with "Add can be found using the extended euclidean algorithm. The pair (n,d) makes up the private key"?

Reply:
Hummm, my bad 😅 

The correct sentence would be _" `d` can be found... "_

Could you adjust the README in English please?

Reply:
No problem 😂

I'll fix this with one more commit